### PR TITLE
Qa modal

### PR DESCRIPTION
**Pull request by hazelkimberly**
Created add question and add answer forms

### CODE REVIEW

File: src/components/related-products-and-comparison/ProductCard.jsx
Comment (jose-m-f) [inline src/components/related-products-and-comparison/ProductCard.jsx:1]: I don't think this change was on purpose but this is my component. Do you think you can take this change out?

Comment (hazelkimberly) [reply src/components/related-products-and-comparison/ProductCard.jsx:1]: Just did! Sorry, not sure how that happened.

**Review by stevkim — APPROVED** [review]
Looks good to me!

File: src/components/related-products-and-comparison/ProductCard.jsx
Comment (hazelkimberly) [reply src/components/related-products-and-comparison/ProductCard.jsx:1]: Just pushed the change!

**Review by jose-m-f — APPROVED** [review]
Looks good

**Pull request merged**